spec/requests/unwithdraw_spec.rb

```diff
@@ -11,104 +11,144 @@
   end
 
   describe "POST /documents/:document/unwithdraw" do
-    let(:withdrawn_edition) { create(:edition, :withdrawn) }
+    let(:edition) { create(:edition, :withdrawn) }
```

Comment:
V minor but in the commit message you mention feature tests when I assume you mean request tests.

Reply:
Thanks - I've fixed that.